packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -698,7 +698,9 @@
             "output": "Output",
             "includerow": "include column name row",
             "newline": "Newline",
-            "usestrings": "parse numerical values"
+            "usestrings": "parse numerical values",
+            "parse_empty_strings": "parse empty strings",
```

Comment:
not sure parse is the right word here - maybe just "include empty strings" ?

Reply:
Done.